typescript/ws-protocol/src/FoxgloveServer.ts

```diff
@@ -375,4 +384,21 @@ export default class FoxgloveServer {
       connection.send(buffer);
     }
   }
+
+  private sendTimeData(connection: IWebSocket, timestamp: bigint): void {
+    const msg = new DataView(new ArrayBuffer(1 + 8));
+    msg.setUint8(0, BinaryOpcode.TIME);
+    msg.setBigUint64(1, timestamp, true);
+
+    // attempt to detect support for {fin: false}
+    if (connection.send.length > 1) {
+      connection.send(msg, { fin: true });
+    } else if (typeof Blob === "function") {
+      connection.send(new Blob([msg]));
+    } else {
+      const buffer = new Uint8Array(msg.byteLength);
+      buffer.set(new Uint8Array(msg.buffer), 0);
+      connection.send(buffer);
+    }
```

Comment:
`fin:false` is not useful here, since we already have all the data in a single buffer, you should be able to use the same API for both web and node and skip the Blob stuff